ui/src/paths.ts

```diff
@@ -8,63 +8,60 @@ const paths = {
   app: {
     root: '/app',
     notifications: '/app/notifications',
-    manage: {
-      root: '/app/manage',
-      custody: '/app/manage/custody',
-      clearing: '/app/manage/clearing',
-      distributions: '/app/manage/distributions',
-      instrument: '/app/manage/instrument',
-      instruments: '/app/manage/instruments',
-      issuance: '/app/manage/issuance',
-      trading: '/app/manage/trading',
-      listings: '/app/manage/listings',
+    distributions: '/app/distributions',
+    instruments: {
+      root: '/app/instruments',
+      instrument: '/app/instruments/instrument',
+      new: {
+        base: '/app/instruments/new/base',
+        convertiblenote: '/app/instruments/new/convertiblenote',
+        binaryoption: '/app/instruments/new/binaryoption',
+      },
     },
+    trading: '/app/trading',
+    listings: { root: '/app/listings', new: '/app/listings/new' },
+    issuance: { root: '/app/issuance', new: '/app/issuance/new' },
     setup: {
       root: '/app/setup',
-      clearing: {
-        offer: '/app/setup/clearing/offer',
-        market: { offer: '/app/setup/clearing/market/offer' },
-      },
-      custody: { offer: '/app/setup/custody/offer' },
-      distribution: {
-        new: { auction: '/app/setup/distribution/new/auction' },
-      },
       identity: '/app/setup/identity',
-      instrument: {
-        new: {
-          base: '/app/setup/instrument/new/base',
-          convertiblenote: '/app/setup/instrument/new/convertiblenote',
-          binaryoption: '/app/setup/instrument/new/binaryoption',
-        },
-      },
-      issuance: { new: '/app/setup/issuance/new' },
-      listing: { new: '/app/setup/listing/new' },
-      trading: { offer: '/app/setup/trading/offer' },
     },
+    // TODO: nest clearing structure
     clearing: {
-      members: '/app/clearing/members',
+      root: '/app/clearing',
       member: '/app/clearing/member',
     },
+    clearingServices: {
+      root: '/app/clearingServices',
```

Comment:
I think the paths would look better (in a browser address bar, for example) with kebab casing, like `/app/clearing-services`...